tests/serialization/serializers/test_serializers.py

```diff
@@ -62,8 +69,17 @@ def test_datetime(self):
         d = deserialize(s)
         assert i.timestamp() == d.timestamp()
 
-    def test_deserialize_datetime_v1(self):
+        i = DateTime(2022, 7, 10, tzinfo=tzutc())
+        s = serialize(i)
+        d = deserialize(s)
```

Comment:
@ferruzzi, IMO your question about encode/decode datetimes from `boto3` covered by this test case

Reply:
I think you are likely right, ad the code does look good "on paper", I was just hoping to make time to actually try it in a live environment to verify.   I won't block the merge if you are confident. :+1: